pkg/sql/plan/function/builtin/binary/generalTime.go

```diff
@@ -0,0 +1,582 @@
+// Copyright 2021 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package binary
+
+import (
+	"fmt"
+	"strings"
+	"unicode"
+)
+
+// GeneralTime is the internal struct type for Time.
+type GeneralTime struct {
```

Comment:
you should rename generalTime.go to general_time.go

Reply:
OK，no problem!